l1-contracts/test/Rollup.t.sol

```diff
@@ -124,6 +125,40 @@ contract RollupTest is RollupBase {
     assertEq(rollup.getProvenBlockNumber(), 1);
   }
 
+  function testSetManaTarget(uint256 _initialManaTarget, uint256 _newManaTarget)
+    public
+    setUpFor("mixed_block_1")
+  {
+    _initialManaTarget = bound(_initialManaTarget, 0, 1e36);
+    _newManaTarget = bound(_newManaTarget, 0, 1e36);
+
+    RollupBuilder builder = new RollupBuilder(address(this));
+    builder.setManaTarget(_initialManaTarget);
+    builder.deploy();
+
+    address governance = address(builder.getConfig().governance);
+    rollup = IInstance(address(builder.getConfig().rollup));
+
+    assertEq(rollup.getManaTarget(), _initialManaTarget);
+
+    // Cannot decrease the mana target
```

Comment:
Best practice for this would really have to just have two tests such that we have that one is failing and another is passing. I know that I was notoriously bad with this in the GSE for doing the same though.